filecoin-proofs/src/bin/paramfetch.rs

```diff
@@ -102,6 +102,19 @@ Defaults to '{}'
                 .long("ipget-bin")
                 .help("Use specific ipget binary instead of looking for (or installing) one in /var/tmp/ipget/ipget"),
         )
+        .arg(
+            Arg::with_name("ipget-args")
```

Comment:
What motivates the existence of this ipget-args argument? What arguments does ipget provide that we want to expose to paramfetch users?

Reply:
From the ipget pr

> Also, add a -peers flag for directly connecting to a set of peers on start.

This will be very useful to speed up connection and downloads